index.js

```diff
@@ -32,24 +25,31 @@ var pify = module.exports = function (fn, P, opts) {
 	};
 };
 
-pify.all = function (obj, P, opts) {
+var pify = module.exports = function (obj, P, opts) {
 	if (typeof P !== 'function') {
 		opts = P;
 		P = Promise;
 	}
 
 	opts = opts || {};
+	opts.exclude = opts.exclude || [/^.+Sync$/];
 
 	var filter = function (key) {
-		if (opts.include) {
-			return opts.include.indexOf(key) !== -1;
-		}
+		var check = function (patterns) {
+			for (var i = 0; i < patterns.length; i++) {
```

Comment:
Sorry, but I don't see the way I can do it.. `[].some` will be pretty useful tho

```js
var filter = function (key) {
	var match = function (pattern) {
		if (typeof pattern === 'string') {
			if (key === pattern) {
				return true;
			}
		} else if (pattern.test(key)) {
			return true;
		}

		return false;
	};

	return opts.include ? opts.include.some(match) : !opts.exclude.some(match);
};
```